src/transformers/models/llama/modeling_llama.py

```diff
@@ -476,15 +476,17 @@ def forward(
         # in fp32. (LlamaRMSNorm handles it correctly)
         input_dtype = query_states.dtype
         if input_dtype == torch.float32:
+            attention_dtype = self.config._flash_attn_2_attention_dtype
```

Comment:
Not all attention modules have a config variable. But guess that's ok we can just pass it forward.